packages/headless/src/api/platform-client.ts

```diff
@@ -109,7 +109,19 @@ export class PlatformClient {
         };
       }
 
-      throw error;
+      if (error.name === 'AbortError') {
```

Comment:
Is there a better way to determine an `AbortError` than the name ? 

Some check with `typeof` maybe ?

Also, is this correctly respected with the polyfill we have to use for locker service ?
See: `getAbortControllerInstanceIfAvailable()`

Reply:
type is an object, simplest differentiator I found was the name property.
Works both with & without polyfill, both throw the same error.